source/content/composer.md

```diff
@@ -50,6 +50,8 @@ A Custom Upstream based off Pantheon's example repositories would need to commit
 
 This workflow has one very serious shortcoming, that is site-specific dependencies are likely to cause a lot of conflicts. The practical use case for this workflow is for a large group of sites that require a single set of dependencies. You should only use this method if you don’t intend to use site specific themes, modules, or plugins downstream.
 
+You can also prevent upstream updates by [setting an empty upstream](/docs/guides/composer-convert/#change-upstreams). This action is available to Site Owner, Organization Administrator, and Users in Charge roles.
```

Comment:
Hmm... I didn't know Org Admins could update them as well. Interesting. Just if there's an Owning Org, or also a Supporting Org Admin? And this has been verified? As far as I knew, only Site Owner or User In Charge could do so.

Reply:
@EdwardAngert please verify or edit.

Reply:
lines up with [Create a Custom Upstream
](https://pantheon.io/docs/create-custom-upstream/) and Product expectations 👍

Reply:
Cool with me. I think I just misread it and thought it meant Supporting Org Admins, which I was surprised about. But now that I look at it again and the other Doc, I get the meaning.